src/styles/page/functions.scss

```diff
@@ -0,0 +1,7 @@
+@function minWidth($rowCardAmount) {
+  @return (200px * $rowCardAmount + 40px * 2 + 48px * ($rowCardAmount - 1));
```

Comment:
What do all these constant pixels mean? Can you optimize calculation?

Reply:
width of each card * amount of cards in row + card-holders paddings + margins between  cards